content/kubeone/master/getting_kubeone/_index.en.md

```diff
@@ -79,13 +79,20 @@ If you have the Go toolchain configured, you can use `go get` to obtain KubeOne.
 To get the latest stable release:
 
 ```
-GO111MODULE=on go get github.com/kubermatic/kubeone
+GO111MODULE=on go get k8c.io/kubeone
```

Comment:
```suggestion
GO111MODULE=on go get k8c.io/kubeone@master
```

before next release this is necessary, otherwise this step will fail

Reply:
Fixed.